### ISSUE TITLE
FOUC on angular.io (possibly related to the dark theme)

### ISSUE THREAD

**Issue by IgorMinar** [angular/angular#42460]
# Docs or angular.io Bug Report

### Description

When the angular.io website is reloaded with an empty cache, I experience FOUC (flash of unstyled content).

This didn't use to be the case until recently and might be related to our implementation of the dark theme #40102 

The issue is present for both light and dark theme.

### What's the affected URL?**
 https://angular.io/

### Reproduction Steps**
- go to angular.io
- refresh with `shift+cmd+r`

## Screenshot
<img width="1542" alt="Screen Shot 2021-06-02 at 4 48 24 PM" src="https://user-images.githubusercontent.com/216296/120565985-15feef00-c3c3-11eb-899b-e013c663733b.png">

### Browser info

desktop Chrome

### Anything else relevant?

Lighthouse or webpagetest.org/test doesn't seem to notice this, which is odd because I can reproduce it via mobile viewport emulation in desktop Chrome.

https://webpagetest.org/result/210602_AiDcGQ_02e8728cf025db44f12402ac41627f27/

**Comment by gkalpak** [comment]
Duplicate of #42365. (Will be fixed by #42435.)

**tl;dr** It's the critical CSS inlining 😞
